privacy-dashboard/privacy-dashboard-impl/src/androidTest/java/com/duckduckgo/privacy/dashboard/impl/animations/TrackersRendererTest.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2022 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.privacy.dashboard.impl.animations
```

Comment:
Can this be a unit test rather than android test?

Reply:
It seems a bit overkill to have one Android test that will need to spin up two emulators in Firebase, etc to test something that seems rather simple.

Reply:
I've moved this to unit test using roboelectric.